### DISCUSSION TITLE
Is there any way i can use cryptosmite on a snappy chromebook?

### DISCUSSION THREAD

**Discussion by darkstorm16** [3kh0/ext-remover#1050] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
im on a snappy chromebook thats on v118 and has kernel 2. i look at the cyptosmite  every week and there no snappy support to it. will it even get a snappy or not and is there another way i can unenroll my device

**Comment by S-PScripts** [comment]
(dead post necropost to answer)

If it still doesn't have snappy support now, probably not

use br1ck
